Description: Merge the two debian/* copyright stanzas
 Two stanzas for the same pattern fail lintian (duplicate-globbing-patterns).

--- a/debian/copyright
+++ b/debian/copyright
@@ -33,13 +33,10 @@
 
 Files: debian/*
 Copyright: 2014-2016 Raoul Gunnar Borenius <borenius@dfn.de>
-License: BSD-2-clause
+           2015-2021 perfSONAR project
+License: BSD-2-clause and Apache-2.0
 
-Files: debian/*
-Copyright: 2015-2021 perfSONAR project
 License: Apache-2.0
-
-License: Apache-2.0
  Licensed under the Apache License, Version 2.0 (the "License");
  you may not use this file except in compliance with the License.
  You may obtain a copy of the License at
